packages/storage/src/components/ImageGallery/hooks/useList.js

```diff
@@ -0,0 +1,27 @@
+import { useState, useCallback } from 'react';
+
+export default function useList(initial) {
+    const [list, set] = useState(() => (Array.isArray(initial) ? initial : []));
```

Comment:
initial is not being used anywhere? why do you keep it there? also if it's internal hook guard logic is unnecessary because you always know the value is being passed